subscriber/podaac_access.py

```diff
@@ -76,11 +109,16 @@ def setup_earthdata_login_auth(endpoint):
     """
     try:
         username, _, password = netrc.netrc().authenticators(endpoint)
-    except (FileNotFoundError, TypeError):
-        # FileNotFound = There's no .netrc file
-        # TypeError = The endpoint isn't in the netrc file,
-        #  causing the above to try unpacking None
-        logging.warning("There's no .netrc file or the The endpoint isn't in the netrc file")
+    # FileNotFound = There's no .netrc file
+    except FileNotFoundError:
+        logging.warning("No .netrc file can be found. One will be attempted to be created.")
+        create_netrc_file(input('Do you have an Earthdata login? (y/n): '))
+        username, _, password = netrc.netrc().authenticators(endpoint)
+    # TypeError = The endpoint isn't in the netrc file,
+    #  causing the below to try unpacking None
+    except TypeError:
+        logging.warning("The endpoint isn't in the netrc file or is incorrect")
+
```

Comment:
I moved all the code from podaac_access to podaac_data_downloader and podaac_data_subscriber. I reverted the podaac_access.py file to what it was before I changed it.

The CLI modules will check if there is no .netrc file. If they do not have an Earthdata login a link to signup for an account will be logged to the console and the CLI tool will quit before a FileNotFound error occurs. Let me know if there is a better or different way you want me to handle this. 

```python
if not netrc_file_exists():
        if input('Do you have an Earthdata login? (y/n): ').lower() == 'y':
            create_netrc_file()
        else:
            logging.info('Go to https://urs.earthdata.nasa.gov/users/new to create an Earthdata login')
            exit()
```

Since the CLI layer now checks for a .netrc file and handles it if there is none can we remove this try except from podaac_access as well? This is the original code for error handling in podaac_access.py before I modified it.

```python
    try:
        username, _, password = netrc.netrc().authenticators(endpoint)
    except (FileNotFoundError, TypeError):
        # FileNotFound = There's no .netrc file
        # TypeError = The endpoint isn't in the netrc file,
        #  causing the above to try unpacking None
        logging.warning("There's no .netrc file or the The endpoint isn't in the netrc file")
```

I am happy to create a flag to ignore a missing .netrc, but want to make sure I get all of this right before doing so.